Monika After Story/game/event-handler.rpy

```diff
@@ -3440,7 +3440,17 @@ label prompts_categories(pool=True):
         # tuplelize the main the category list
         # NOTE: we use a 2nd list here to do displaying, keeping track of the
         # older cat list for checking if a category was picked
-        dis_cat_list = [(x.capitalize() + "...",x) for x in main_cat_list]
+        dis_cat_list = []
+        for x in main_cat_list:
+            if x.lower() == "ddlc":
+                capitalized_string = x.upper() + "..."
+            elif x.lower () == "literature club":
+                capitalized_string = "Literature Club..."
+            else:
+                capitalized_string = x.capitalize() + "..."
```

Comment:
single-catch cases like this should be avoided in favour of more generic, reusable solutions.

Perhaps it's best to remove the capitalization function entirely in categories and simply make events use the capitalization they require.

Or to help deal with typos/duplicate categories down the line, given the move to renpy 8 in the future, maybe we could use enums?

Reply:
> Perhaps it's best to remove the capitalization function entirely in categories and simply make events use the capitalization they require.

![image](https://user-images.githubusercontent.com/96905447/192122177-8688ea1f-a687-4ddf-9c12-ee52d7b7fbec.png)
By that do you mean capitalizing these in the event information for each topic? Sorry if I am misunderstanding.

Reply:
Yes. However before you do that, in the interest of saving you the time/effort in case this is no'd: @ThePotatoGuy, what are your thoughts here?

Reply:
Can we please go the enum approach? We had problems before when we missed a typo in a category. Right now you could just use a static class and later on replace with an enum when we get r8.

Reply:
This is what I was thinking too. Definitely want enums here as it'd make it soooo much cleaner to migrate too.

Reply:
Very sorry. I do not understand enum too well. Does this mean a new class that has manually specified capitalization values for each category that exists, or something else?

Reply:
enum approach is fine for py3 as long as we support basic strings in the category list, too.